br/pkg/lightning/backend/external/engine.go

```diff
@@ -227,13 +324,8 @@ func (e *Engine) SplitRanges(
 	return ranges, nil
 }
 
-// Close releases the resources of the engine.
-func (e *Engine) Close() error {
-	if e.iter == nil {
-		return nil
-	}
-	return errors.Trace(e.iter.Close())
-}
+// Close implements common.Engine.
+func (e *Engine) Close() error { return nil }
```

Comment:
Doube confirm, external engine close do not need release any resource, right?

Reply:
For the rest lgtm.

Reply:
yes, local engine need to close its pebble DB. And before this PR external engine holds an iterator member that needs to be closed, but in this PR the iterator becomes a local variable that closed in line 144.